ofctrl/ofctrl_test.go

```diff
@@ -73,15 +73,11 @@ func (o *OfActor) TLVMapReplyRcvd(sw *OFSwitch, tlvTableStatus *TLVTableStatus)
 	}
 }
 
-var ofActor OfActor
```

Comment:
can you move the test changes to a different PR?
are these changes related to https://github.com/antrea-io/ofnet/issues/2?